TypeCobol.Analysis/Cfg/ControlFlowGraphBuilder.CfgSentence.cs

```diff
@@ -0,0 +1,77 @@
+﻿using System.Collections.Generic;
+using TypeCobol.Compiler.Symbols;
+
+namespace TypeCobol.Analysis.Cfg
+{
+    public partial class ControlFlowGraphBuilder<D>
+    {
+        /// <summary>
+        /// A Sentence used for our builder. A Sentence is a special symbol.
+        /// </summary>
+        internal class CfgSentence : Symbol
+        {
+            /// <summary>
+            /// First Block associated to this sentence.
+            /// </summary>
+            internal BasicBlockForNode Block
```

Comment:
=> Allblocks.First() ?

Reply:
No it is the block of the sentence, in fact in COBOL the basic block unit is the Sentence, (in other words it is the dot character separator).

Reply:
See PR #1738.